src/hooks/project/usePaymasterAttest.tsx

```diff
@@ -54,11 +54,14 @@ const usePaymasterAttest = () => {
         external_url: externalUrl,
         credits,
       })
+      const author = credits.map((credit: any) => credit.name)
```

Comment:
nit: should be plural variable name
```suggestion
      const authors = credits.map((credit: any) => credit.name)
```